eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -125,6 +127,7 @@ class SqliteAuditDb(sqlite: Connection) extends AuditDb {
       var q: Queue[PaymentSent] = Queue()
       while (rs.next()) {
         q = q :+ PaymentSent(
+          id = ChannelCodecs.UNKNOWN_UUID, // TODO: temporary! fix that
```

Comment:
We should add a payment id column to the audit.sent table. Or maybe just the payment hash?